uptane-standard.md

```diff
@@ -147,15 +147,42 @@ A standards document that can guide the safe design, integration and deployment
 
 ## Scope of Standards Coverage
 
-TODO
+This document sets guidelines for implementing Uptane in most systems capable of updating software on connected units in cars. In this section, we define the scope of that applicability by providing sample use cases and possible exceptions, aspects of update security that are not applicable to Uptane, and the design requirements governing the preparation of these standards.
 
 ### Use Cases
 
-TODO
+The following use cases provide a number of scenarios illustrating the manner in which software updates could be accomplished using Uptane.
+
+#### OEMs initializing Uptane at the factory using SOTA
+
+Bob, who works for an OEM, is overseeing the installation of Uptane on new vehicles at a manufacturing plant. He starts with preparing the ECUs by adding the following components: code to perform full and partial verification, the latest copy of the relevant metadata, the public keys, and the latest time, signed by the time server. His implementation would be considered Uptane-compliant if:
```

Comment:
Is it valuable to use individuals names for our usecases? Most of these things wouldn't be done by a sole individual and is many of the cases the "person" is actually a large corporation with many people involved. I'm questioning the utility of this, versus identifying having identified roles such as OEM, Tier1, etc.

Reply:
I was going by the examples I had, which used names. I'm fine with switching it to just roles.